cmd/kube-apiserver/app/options/options.go

```diff
@@ -173,7 +174,8 @@ func (s *ServerRunOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.IPNetVar(&s.ServiceClusterIPRange, "service-cluster-ip-range", s.ServiceClusterIPRange, ""+
 		"A CIDR notation IP range from which to assign service cluster IPs. This must not "+
-		"overlap with any IP ranges assigned to nodes for pods.")
+		"overlap with any IP ranges assigned to nodes for pods. [default='"+
+		services.DefaultServiceIPCIDR+"'.")
```

Comment:
No need to break into 2 lines, since the string is not quite long. Also this will break the readability when strings contain `'` and `"`.